tasks/build.js

```diff
@@ -64,10 +59,7 @@ async function run() {
     const args = process.argv.slice(2);
 
     if (args.includes('--release')) {
-        await release();
-    }
-    if (args.includes('--debug')) {
-        await debug({watch: args.includes('--watch')});
+        await release({watch: args.includes('--watch'), debug: args.includes('--debug')});
```

Comment:
Now there's a dilemma. Let's use `--release` (`build({debug: false, watch: false})`) and `--debug` (`build({debug: true, watch: args.includes('--watch')})`) separately.